halo2_gadgets/src/sinsemilla.rs

```diff
@@ -329,6 +343,23 @@ where
             .map(|(point, zs)| (ecc::NonIdentityPoint::from_inner(self.ecc_chip.clone(), point), zs))
     }
 
+    #[allow(non_snake_case)]
+    #[allow(clippy::type_complexity)]
+    /// $\mathsf{SinsemillaHashToPoint}$ from [§ 5.4.1.9][concretesinsemillahash].
+    ///
+    /// [concretesinsemillahash]: https://zips.z.cash/protocol/protocol.pdf#concretesinsemillahash
```

Comment:
This link does not describe the functionality, it should point into the relevant section in the zsa zips. If at all.

Reply:
Updated comment by describing the function